src/wexond/renderer/app/components/Bubble/index.tsx

```diff
@@ -25,18 +27,19 @@ export const Bubble = ({
   disabled,
   onCloseClick,
 }: {
-  children?: any;
-  icon?: string;
-  invert?: boolean;
-  maxLines?: number;
-  iconSize?: number;
-  onClick?: (e?: React.MouseEvent<HTMLDivElement>) => void;
-  onCloseClick?: (e?: React.MouseEvent<HTMLDivElement>) => void;
-  itemsPerRow?: number;
-  disabled?: boolean;
-  toggled?: boolean;
-}) => {
-  const width = getSize(itemsPerRow);
+  children?: any
```

Comment:
```suggestion
  children?: JSX.Element
```

OR 
```suggestion
 export const Bubble = React.FC<{icon?: string
  invert?: boolean
  maxLines?: number
  iconSize?: number
  onClick?: (e?: React.MouseEvent<HTMLDivElement>) => void
  onCloseClick?: (e?: React.MouseEvent<HTMLDivElement>) => void
  itemsPerRow?: number
  disabled?: boolean
  toggled?: boolean}> = ....
```

Reply:
React.FC does not seem to be available in this version